refresh-dashboard.rb

```diff
@@ -29,8 +29,8 @@
 
   def xml2html(context, xmlFiles, xsltFile, outputDirectory)
     Dir.glob(xmlFiles).each do |inputFile|
-      stylesheet = LibXSLT::XSLT::Stylesheet.new( LibXML::XML::Document.file(xsltFile))
       xml_doc = LibXML::XML::Document.file(inputFile)
+      stylesheet = LibXSLT::XSLT::Stylesheet.new( LibXML::XML::Document.file(xsltFile))
```

Comment:
Any reason for this change? i.e. a nefarious bug?

Reply:
No. Only me forgetting to revert things I changed while debugging a weird combination of Ubuntu/ xslt/ ruby:

https://github.com/xml4r/libxslt-ruby/pull/13

https://github.com/tonyfg/libxslt-ruby/pull/1